contentreview-impl/pack/src/webapp/WEB-INF/components.xml

```diff
@@ -76,13 +76,14 @@
 	-->
 	
 	<!--  Uncomment this to use a global property to define if every site uses c-r -->
-	<!--<bean id="org.sakaiproject.contentreview.service.ContentReviewSiteAdvisor" class="org.sakaiproject.contentreview.impl.advisors.GlobalPropertyAdvisor">
+	<bean id="org.sakaiproject.contentreview.service.ContentReviewSiteAdvisor" class="org.sakaiproject.contentreview.impl.advisors.GlobalPropertyAdvisor">

```

Comment:
The global advisor, using the cutover date, is the simplest way forward. Admins don't have to worry about setting properties on every site to determine which assignments using which integration, it is all automatic. To me this makes more sense as the default.